packages/drawnix/src/utils/image.spec.ts

```diff
@@ -0,0 +1,92 @@
+// SVG Export Tests
+// Testing utility functions for SVG export functionality
+
+describe('SVG Export Utility Functions', () => {
```

Comment:
I'm concerned that our test cases are too isolated from real business logic and now unit testing is unnecessary.